packages/vscode-extension/messages.json

```diff
@@ -83,6 +83,7 @@
 	"IMPORT_FILES_QUESTIONS_SELECT_FOLDER": "Select the path from where you want to import the files.",
 	"IMPORT_FILES_QUESTIONS_OVERRIDE": "The selected files in the project will be overwritten.",
 	"IMPORT_FILES_QUESTIONS_OVERRIDE_SINGLE": "The selected file in the project will be overwritten: {0}",
+	"IMPORT_FILES_UPDATING_FILE": "Updating file...",
```

Comment:
Following the previous comment from ImportFilesAction.js this message probably also needs to be deleted.